### ISSUE TITLE
Massive discrepancies in USD volume outputs from identical dex.trades-based queries today vs. one month ago

### ISSUE THREAD

**Issue by wkueshner** [duneanalytics/spellbook#6351]
### Description
 I'm getting very different results than I was approx. 25 days ago for a dex.trades-based query that calculates historical cumulative volume deltas (CVDs) given a wallet address or set of addresses as input. Nothing changed about the query in question, just the result set. *The unified_trades CTE is the relevant portion; I have verified that outputs related to the other CTEs have not changed.*

Which leads me to my question: Did some DEXs break and/or get temporarily offboarded during the recent DEX sector reorganization? Or, to the contrary, did `dex.trades` recently get injected with a huge backfill of historical volume? The biggest CVD "changes" (largely "flatlines" i.e. disappearances of trading volumes that were previously in Dune's database as far as I can tell) seem to be of the Q42021-2022 DEX trading vintage.

Below is a visual reflection of the before/after difference in the results sets. (The blue lines/left axes reflect the query's results, i.e. the entity's collective CRV and UNI CVDs in USD terms, respectively. The orange areas are from a separate query & not relevant.

**CRV, Previous result —**
<img width="1383" alt="CRV Cumulative Volume Delta 6-20-2024" src="https://github.com/user-attachments/assets/68902ca6-4f03-427a-a57c-6e9149098ccb">
**CRV, New result —**
<img width="1070" alt="CRV Cumulative Volume Delta 7-9-2024" src="https://github.com/user-attachments/assets/04cad42a-f692-440a-9655-6454fcb9dd30">
Difference: a $130 million (65%) reduction in CVD, from what looks to be less volume altogether, not more volume in the sell direction. This looks to be true in all of the before/after screenshots I have saved.

Another example:

**UNI, Previous result —**
<img width="1383" alt="UNI Cumulative Volume Delta 6-20-2024" src="https://github.com/user-attachments/assets/839c2209-38b6-445a-b916-608ac08cfda7">
**UNI, New result —**
<img width="1070" alt="UNI Cumulative Volume Delta 7-9-2024" src="https://github.com/user-attachments/assets/d925f6cd-fdb3-4a80-b6d9-444835d86419">
Difference: a ~$70 million (40%) reduction in CVD, from what again looks to be less trading volume altogether rather than a newly integrated dex.trades source of sell volume.

The trades being analyzed all already had populated amount_usd fields, so the issue isn't amount_usd coverage either.

### Impacted model(s)
`dex.trades`

**Comment by 0xBoxer** [comment]
Hey @wkueshner, please provide us with the query ids so we can look at execution history and debug from there.

**Comment by wkueshner** [comment]
Hey @0xBoxer, these Query IDs that should help: 3700834 (captures the all-time CVD for one specific token traded by a given cluster) and 3740523 (captures CVDs for each token traded by a given cluster). I know that both of these were run at least once before (in May) and after (in July) the change.

**Comment by jeff-dude** [comment]
thanks @0xBoxer for initiating the response, let us know if we can help here 🙏

**Comment by 0xBoxer** [comment]
hey @wkueshner, sorry for the delay, busy times in spellbook currently.

After riddling over this for a bit, here is my most likely theories:
lines 81-84 in [query_3700834](https://dune.com/queries/3700834) is excluding trades based on if they show up in aggregator trades. If we added new aggregator trades between may and now and have retroactively indexed new data, this might just have contributed to a decrease in volume. 

```sql
...
        AND NOT EXISTS (
            SELECT 1
            FROM dex_aggregator.trades t
            WHERE t.tx_hash = evms.erc20_transfers.evt_tx_hash AND (t.taker IN ({{MM wallets comma-separated}}) OR t.maker IN ({{MM wallets comma-separated}}) OR t.tx_from IN ({{MM wallets comma-separated}}))
...            
```

Additionally, we also changed some of our prices logic and retroactively changed some `amount_usd` calcs, but I'd need to do more investigation on the timelines for that to see if that could be the cause here.

Does my `dex_aggregator.trades` upstream changes theory make any sense to you? I am not sure I full context of this query, so you'll be the better judge of this.
